docs/import/manage_layers.js

```diff
@@ -328,31 +327,10 @@ function populateStateAndDisasterAssetPickers(disaster) {
   for (const state of states) {
```

Comment:
do we need this loop? And the code right above it?

Reply:
I don't think so, I think the only promise that needs to be returned here is from disasterDone if that exists